src/components/plots/plot.ts

```diff
@@ -270,20 +270,23 @@ module Plottable {
       this._scales().forEach((scale) => scale._autoDomainIfAutomaticMode());
     }
 
-    private _updateExtentsForAttr(attr: string) {
+    protected _updateExtentsForAttr(attr: string, filter?: _Accessor) {
       var binding = this._attrBindings.get(attr);
       var accessor = binding.accessor;
       var extents = this._datasetKeysInOrder.map((key) => {
         var plotDatasetKey = this._key2PlotDatasetKey.get(key);
         var dataset = plotDatasetKey.dataset;
         var plotMetadata = plotDatasetKey.plotMetadata;
-        return this._computeExtent(dataset, accessor, plotMetadata);
+        return this._computeExtent(dataset, accessor, plotMetadata, filter);
```

Comment:
Err I'm unsure what you're meaning by a map of filters, but I was referring that this `filter` argument here should not be inputted via an argument but instead through possibly like a `getFilterForAttr()` protected endpoint